app/styles/partials/footer.scss

```diff
@@ -2,6 +2,9 @@
   display: flex;
   min-height: 100vh;
   flex-direction: column;
+  .main-container {
```

Comment:
For mobile screens, where the middle containers takes up the full width, we want the rail menu to be hidden and instead a different menu at the top of the content. 
This can be dealt with as separate issue. 

In this issue, you can just fix for the other screen sizes where we still can have rails.